packages/models-library/src/models_library/rest_pagination_utils.py

```diff
@@ -69,9 +70,11 @@ def paginate_data(
     """
     last_page = ceil(total / limit) - 1
 
+    data = [item.model_dump() if hasattr(item, "model_dump") else item for item in chunk]
+
     return PageDict(
         _meta=PageMetaInfoLimitOffset(
-            total=total, count=len(chunk), limit=limit, offset=offset
+            total=total, count=len(data), limit=limit, offset=offset
         ),
```

Comment:
not sure I understand that change. chunk is a list, so why does it need a change?